apps/admin-x-activitypub/src/components/Activities.tsx

```diff
@@ -164,19 +176,55 @@ const Activities: React.FC<ActivitiesProps> = ({}) => {
 
     const maxAvatars = 5;
 
+    const {data: userProfile, isLoading: isLoadingProfile} = useUserDataForUser(user) as {data: ActorProperties | null, isLoading: boolean};
+
     const {getActivitiesQuery} = useActivitiesForUser({
         handle: user,
         includeOwn: true,
         includeReplies: true,
         filter: {
-            type: ['Follow', 'Like', `Create:Note:isReplyToOwn`]
```

Comment:
`isReplyToOwn` doesn't appear to be working correctly, and will be removed when we switch to a dedicated notifications endpoint